docs/resources/obs_bucket.md

```diff
@@ -1,352 +1,352 @@
+  format bucket policy,
+  see the [Developer Guide](https://support.huaweicloud.com/intl/en-us/perms-cfg-obs/obs_40_0004.html).
+
+* `policy_format` - (Optional, String) Specifies the policy format, the supported values are *obs* and *s3*. Defaults
+  to *obs*.
+
+* `tags` - (Optional, Map) A mapping of tags to assign to the bucket. Each tag is represented by one key-value pair.
+
+* `versioning` - (Optional, Bool) Whether enable versioning. Once you version-enable a bucket, it can never return to an
+  unversioned state. You can, however, suspend versioning on that bucket.
+
+* `logging` - (Optional, Map) A settings of bucket logging (documented below).
+
+<!-- markdownlint-disable MD033 -->
+
+* `quota` - (Optional, Int) Specifies bucket storage quota. Must be a positive integer in the unit of byte. The maximum
+  storage quota is 2<sup>63</sup> – 1 bytes. The default bucket storage quota is 0, indicating that the bucket storage
+  quota is not limited.
+
+<!-- markdownlint-enable MD033 -->
+
+* `website` - (Optional, List) A website object (documented below).
+
+* `cors_rule` - (Optional, List) A rule of Cross-Origin Resource Sharing (documented below).
+
+* `lifecycle_rule` - (Optional, List) A configuration of object lifecycle management (documented below).
+
+* `force_destroy` - (Optional, Bool) A boolean that indicates all objects should be deleted from the bucket, so that the
+  bucket can be destroyed without error. Default to `false`.
+
+* `region` - (Optional, String, ForceNew) Specifies the region where this bucket will be created. If not specified, used
+  the region by the provider. Changing this will create a new bucket.
+
+* `parallel_fs` - (Optional, Bool, ForceNew) Whether enable a bucket as a parallel file system. Changing this will
+  create a new bucket.
+
+* `multi_az` - (Optional, Bool, ForceNew) Whether enable the multi-AZ mode for the bucket. When the multi-AZ mode is
+  enabled, data in the bucket is duplicated and stored in multiple AZs.
+
+  -> **NOTE:** Once a bucket is created, you cannot enable or disable the multi-AZ mode. Changing this will create a new
+  bucket, but the name of a deleted bucket can be reused for another bucket at least 30 minutes after the deletion.
+  Exercise caution when changing this field.
+
+* `encryption` - (Optional, Bool) Whether enable default server-side encryption of the bucket in SSE-KMS mode.
+
+* `kms_key_id` - (Optional, String) Specifies the ID of a KMS key. If omitted, the default master key will be used.
+
+* `kms_key_project_id` - (Optional, String) Specifies the project ID to which the KMS key belongs. If omitted, the ID
+  of the provider-level project will be used.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project id of the OBS bucket.
+Default to `0`.
+
+The `logging` object supports the following:
+
+* `target_bucket` - (Required, String) The name of the bucket that will receive the log objects. The acl policy of the

```

Comment:
the param order should adjusted to: required, forcenew, optional

Reply:
format error, I have convert it with dos2unix